Dockerfile.libp2p

```diff
@@ -1,6 +1,6 @@
 # BUILD IMAGE ---------------------------------------------------------
```

Comment:
Why are we renaming this `Dockerfile`?

Reply:
We are about to merge the mixnet node that will be used alongside libp2p node. The idea was to keep those two versions in a separate docker files: `Dockerfile.libp2p` and `Dockerfile.mixnet`. The build image will be the same, but the final binaries in final images will differ.

Reply:
The usual practice for that is to have subfolders named `libp2p` and `mixnet` with `Dockerfile` inside.